### PR TITLE
Fix gpt2 link

### PR DESCRIPTION
**Pull request by brendan-ai2**
- Frontend link only

### CODE REVIEW

**Comment by brendan-ai2** [comment]
This works for me locally, but my frontend experience is very limited, so I don't know if it's the "right way". Took me a bit too as I started by modifying `app.py` which I now think isn't necessary. @Eric-Wallace, do you rely on linking to the backend or just the frontend? I.e., are you hitting us programmatically or just linking the site itself?

**Comment by joelgrus** [comment]
I would prefer a less one-off solution. something along the lines of creating `redirects.js` that looks like

```
{"gpt2": "next_model_lm"}
```

and then you could load it in in `App.js`, and in the top level `Demo` component check it and remap the model if necessary. or something similar like that, does that make sense?

**Comment by brendan-ai2** [comment]
Yep, makes sense. I've done something similar by including a `redirects` field in `demo/src/models.js`. Given that external users have noticed I don't think we need a perfect solution here, but let me know if I've done something very stupid.

File: demo/src/App.js [hunk 1]
```diff
@@ -47,12 +47,14 @@ const App = () => (
 // and it renders the specific task in an iframe.
 const Demo = (props) => {
   const { model } = props.match.params
+  const redirectedModel = modelRedirects[model] ? modelRedirects[model] : model
```
Comment (joelgrus) [inline demo/src/App.js]: simpler: `modelRedirects[model] || model`

File: demo/src/App.js [hunk 2]
```diff
@@ -47,12 +47,14 @@ const App = () => (
...
+  props.match.params.model = redirectedModel
```
Comment (joelgrus) [inline demo/src/App.js]: modifying `props` like this seems bad, instead I'd do (see next comment)

File: demo/src/App.js [hunk 3]
```diff
@@ -47,12 +47,14 @@ const App = () => (
...
 
   return (
     <React.Fragment>
       <Header alwaysVisible={true} />
       <div className="pane-container">
-        <Menu selectedModel={model} clearData={() => {}}/>
+        <Menu selectedModel={redirectedModel} clearData={() => {}}/>
         <SingleTaskFrame {...props}/>
```
Comment (joelgrus) [inline demo/src/App.js]: `<SingleTaskFrame {...props, model: redirectedModel}/>

Comment (brendan-ai2) [reply demo/src/App.js]: This doesn't seem to work.

```
./src/App.js
  Line 57:  Parsing error: Unexpected token, expected "}"

  55 |       <div className="pane-container">
  56 |         <Menu selectedModel={redirectedModel} clearData={() => {}}/>
> 57 |         <SingleTaskFrame { ...props, model: redirectedModel }/>
     |                                    ^
  58 |       </div>
  59 |     </React.Fragment>
  60 |   )
```

Comment (brendan-ai2) [reply demo/src/App.js]: `model` isn't a parameter to `SingleTaskFrame`. I tried adding it but that didn't seem to help. Pushed the broken code so you can see what I tried.

Comment (joelgrus) [reply demo/src/App.js]: oh, I see, then we'll have to tweak SingleTaskFrame to do

```
  const { slug } = props.match.params
  const { model } = props
```

this is not ideal, but I believe having a component modify its props is a react antipattern, cf https://stackoverflow.com/questions/24939623/can-i-update-a-components-props-in-react-js

**Comment by brendan-ai2** [comment]
Thanks for the help and review!

**Pull request merged**